skins/item/color/yellow.ess

```diff
@@ -0,0 +1,76 @@
+background = __.color.yellow.light
+color hsla(49.6, 73.1%, 26.3%, 1)
+
+.item-label,
+&-main,
+&-label
+  &:after
+    border-color hsla(49.6, 73.1%, 26.3%, 1)
+    
+&:hover
+  color hsla(49.6, 73.1%, 26.3%, 1)
+  
+&:hover > &-main,
+&:active > &-main,
+&.is-active > &-main,
+&:hover > &-content &-label,
+&:active > &-content &-label,
+&.is-active > &-content &-label,
+  &:after
+    border-color hsla(49.6, 73.1%, 26.3%, 1)
+    
+&-input:focus + &-label
+  color hsla(49.6, 73.1%, 26.3%, 1)
+  &:after
+    border-color hsla(49.6, 73.1%, 26.3%, 1)
+    
+// doesn't use variable because the placeholder hijacks it for some reason
+&-input
+  color hsla(49.6, 73.1%, 26.3%, 1)
```

Comment:
Yeah, I saw the comment, but that's not cool. We need to find out why it's happening and fix it.  Hard colors are no bueno